godot/Demos/PaletteSwap2D.tscn

```diff
@@ -0,0 +1,65 @@
+[gd_scene load_steps=8 format=2]
```

Comment:
The file name and its root node should be PaletteSwapDemo to be consistent with the rest of the demos.
Same for the GDScript file.